Framework/PythonInterface/plugins/CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ set(PYTHON_PLUGINS
   algorithms/WorkflowAlgorithms/IndirectReplaceFitResult.py
   algorithms/WorkflowAlgorithms/ILL_utilities.py
   algorithms/WorkflowAlgorithms/D7YIGPositionCalibration.py
+  algorithms/WorkflowAlgorithms/D7AbsoluteScaleNormalisation.py
```

Comment:
So at the end this does component separation, detector efficiency and absolute scale, so I think the best name would be `D7AbsoluteCrossSections`; it kind of tells what is the output.